packages/firestore/rollup.shared.js

```diff
@@ -129,7 +129,8 @@ const manglePrivatePropertiesOptions = {
   },
   mangle: {
     properties: {
-      regex: /^__PRIVATE_/
+      regex: /^__PRIVATE_/,
+      reserved: ['do']
```

Comment:
Should we just add all JS keywords here? This will fail with `if` and `of` next and it is really tough to figure out these failures.

Reply:
Done.